recipes/opencv/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(OpenCV REQUIRED CONFIG)
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package opencv::opencv_imgproc opencv::opencv_highgui opencv::opencv_objdetect)
+set_property(TARGET test_package PROPERTY CXX_STANDARD 11)
```

Comment:
Shouldn't be necessary for opencv < 4

Reply:
I had an error without c++11, I need to check what required it.